src/views/ModelPlan/CRTDL/AddCRTDL/index.tsx

```diff
@@ -296,8 +295,8 @@ const AddCRTDL = () => {
                               <div className="width-card-lg position-relative">
                                 <Field
                                   as={DatePicker}
-                                  error={+!!flatErrors.dateInitiated}
                                   id="cr-tdl-date-initiated"
+                                  error={+!!flatErrors.dateInitiated}
```

Comment:
is this supposed to be like this?
I see all the others are being deleted.

Reply:
Yes that's supposed to be there